examples/AWSDriverExample/src/main/java/software/amazon/OktaAuthPluginExample.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import software.amazon.jdbc.PropertyDefinition;
+import software.amazon.jdbc.plugin.federatedauth.FederatedAuthPlugin;
+import software.amazon.jdbc.plugin.federatedauth.OktaAuthPlugin;
+
+public class OktaAuthPluginExample {
+
+  private static final String CONNECTION_STRING = "jdbc:aws-wrapper:postgresql://database.cluster-xyz.us-east-1.rds.amazonaws.com:5432/postgres";
+
+  public static void main(String[] args) throws SQLException {
+    // Set the Okta Authentication Connection Plugin parameters and the JDBC Wrapper parameters.
+    final Properties properties = new Properties();
+
+    // Enable the Okta Authentication Connection Plugin.
+    properties.setProperty(PropertyDefinition.PLUGINS.name, "okta");
+    properties.setProperty(OktaAuthPlugin.IDP_ENDPOINT.name, "123456789.okta.com");
+    properties.setProperty(OktaAuthPlugin.IDP_NAME.name, "okta");
```

Comment:
I don't think we really need the IDP_NAME parameter. All the authentication methods will be enabled and implemented using Plugins so it is a bit weird to have another parameter explicitly stating the IDP name. @sergiyvamz @aaronchung-bitquill thoughts on deprecating it?

Reply:
I see that IDP_NAME is used to build a saml url. Is there any chance that IDP_NAME may have value other than "okta"?

Reply:
Okta does not need idp name in the SAML url so I think we are good